Session9/src/com/itschool/homework/Session9Challenge4.java

```diff
@@ -0,0 +1,20 @@
+package com.itschool.homework;
+
+import java.util.Scanner;
+
+public class Session9Challenge4 {
+    public static void main(String[] args) {
+        Session9Challenge4 run = new Session9Challenge4();
+        run.areDatesEqual();
+    }
+
+    public void areDatesEqual() {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Please insert 1st date in format YYYY-MM-DD");
+        String firstDate = input.nextLine();
+        System.out.println("Please insert 2nd date in format YYYY-MM-DD");
+        String secondDate = input.nextLine();
+        System.out.println("Both dates are equal?");
+        System.out.println(firstDate.equals(secondDate));
```

Comment:
this will work just fine, but you are using Strings. The purpose of the exercise is to use dates objects. So, could you convert these 2 objects to `LocalDate` or so and then comparing them? 
Hint: you can use the method `parse` that you already used in other exercise and then compare the dates with `isEqual` method.